### PR TITLE
Fix bug where "Yes"-votes are prevented when attacked by phantoms, even with `phantom-support=false`

### LINKED ISSUE #16: Players can't vote "yes" if they're attacked by phantoms, even if "phantom-support=false"
Attempting to run this plugin on my server, I noticed, that the `phantom-support=false`-rule is limited to starting new votes:
image

With `phantom-support=false`, players are able to start a vote using `/skipnight`, but, if another player has started the vote and they attempt to vote `YES`, they're unable to do so.

### PR DESCRIPTION
**Pull request by ViMaSter**
This closes https://github.com/mattboy9921/SkipNight/issues/16

### CODE REVIEW

File: src/main/java/net/mattlabs/skipnight/Vote.java [hunk 1]
```diff
@@ -279,7 +279,13 @@ public void addYes(UUID uuid, VoteType voteType) {
             if (voters.contains(voter)) {
                 voter = voters.get(voters.lastIndexOf(voter));
                 if (voter.getVote() == 0) {
-                    if (this.voteType == VoteType.NIGHT && Bukkit.getPlayer(uuid).getStatistic(Statistic.TIME_SINCE_REST) > 72000) {
+                    boolean playerMustSleep;
+                    // Check version for TIME_SINCE_REST added in 1.13
+                    if (Versions.versionCompare("1.13.0", version) <= 0) {
+                        playerMustSleep = player.getStatistic(Statistic.TIME_SINCE_REST) >= 72000;
```
Comment (mattboy9921) [inline src/main/java/net/mattlabs/skipnight/Vote.java]: This is checking the statistic for the player who started the vote, not the player who just voted. It should be:
```java
Bukkit.getPlayer(uuid).getStatistic(Statistic.TIME_SINCE_REST) >= 72000;
```

File: src/main/java/net/mattlabs/skipnight/Vote.java [hunk 2]
```diff
@@ -303,7 +309,13 @@ public void addNo(UUID uuid, VoteType voteType) {
...
```
Comment (mattboy9921) [inline src/main/java/net/mattlabs/skipnight/Vote.java]: This is checking the statistic for the player who started the vote, not the player who just voted. It should be:
```java
Bukkit.getPlayer(uuid).getStatistic(Statistic.TIME_SINCE_REST) >= 72000;
```

**Review by mattboy9921 — CHANGES_REQUESTED** [review]
Just need to fix what player is referenced for the statistic.

File: src/main/java/net/mattlabs/skipnight/Vote.java (hunk 1 above)
Comment (ViMaSter) [reply src/main/java/net/mattlabs/skipnight/Vote.java]: Ah, good catch, thanks for pointing that out! Addressed it in a1a0c8b.

**Review by mattboy9921 — APPROVED** [review]
Looks good.

**Pull request merged**